packages/expo-optimize/src/assets.ts

```diff
@@ -17,6 +17,14 @@ import { basename, join, parse, relative } from 'path';
 import prettyBytes from 'pretty-bytes';
 import temporary from 'tempy';
 
+function fileExists(file: string): boolean {
```

Comment:
why the different implementations here and in `fileExistsAsync`? i understand the code between expo-cli utils and expo-optimize isn't shared, but probably no need for an async version to exist and this sync version is easier to read/understand imo.